code/src/lib/Button.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
```

Comment:
Really good that you're so consistent with your styling, that you use styled components and that you have structured them using these lib/ folders 🌟